Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -529,7 +535,7 @@ public struct FileRuleDescription {
         .init(
             rule: .compile,
             toolsVersion: .minimumRequired,
-            fileTypes: ["swift"]
+            fileTypes: ["swift", "md", "rst", "tex"]
```

Comment:
The new file types should get their own distinct rule with tools-version being 5.6